randovania/games/fusion/logic_database/Sector 2 TRO.txt

```diff
@@ -258,34 +258,41 @@ Extra - room_id: [9]
               # Stand on Frozen Reo
               Combat (Beginner) and Stand On Frozen Enemies (Beginner) and Can Freeze Enemies
   > Door to Maintenance Annex
+      After Sector 2 Zoro Zig-Zag Bomb Blocks Broken and Can Bounce in Ball
+  > Event - Break Bomb Blocks
       Can Bounce in Ball and Can Use Any Bombs
 
 > Door to Maintenance Annex; Heals? False
   * Layers: default
   * L0 Hatch to Maintenance Annex/Door to Zoro Zig-Zag
   * Extra - door_idx: (21,)
   > Door to Data Hub
+      All of the following:
+          Morph Ball and After Sector 2 Zoro Zig-Zag Bomb Blocks Broken
+          Any of the following:
+              Space Jump
+              Hi-Jump and Wall Jump (Intermediate)
+  > Event - Break Bomb Blocks
       All of the following:
           Morph Ball
           Any of the following:
-              All of the following:
-                  Space Jump
-                  Any of the following:
-                      Power Bombs
-                      All of the following:
-                          Morph Ball Bomb Data
-                          Any of the following:
-                              Jump Bombjump (Expert)
-                              Screw Attack and Knowledge (Beginner)
-              All of the following:
-                  Hi-Jump
-                  Any of the following:
-                      Power Bombs
-                      All of the following:
-                          Morph Ball Bomb Data
-                          Any of the following:
-                              Jump Bombjump (Advanced)
-                              Screw Attack and Knowledge (Beginner) and Wall Jump (Intermediate)
+              # Climbing Requirement
+              Space Jump
+              Hi-Jump and Wall Jump (Intermediate)
+          Any of the following:
+              # Break Blocks
+              Can Use Power Bombs
+              Screw Attack and Knowledge (Beginner)
+              # Cursed Method: https://youtu.be/Cqf5mZg1NZc
+              Hi-Jump and Jump Bombjump (Hypermode) and Mid-Air Morph (Intermediate) and Can Use Bombs
+
+> Event - Break Bomb Blocks; Heals? False
```

Comment:
Which bomb block is this referring to. All?
Also why is Bounce In Ball required? Looking at the map, doesnt look like you need that to *break* the blocks. Not even to leave.

Reply:
yes, this is referring to all the blocks. thought about having each one be independent but decided against it since you gain nothing from having only some of them broken.